app/src/main/res/values/strings.xml

```diff
@@ -433,6 +433,13 @@
     <string name="resize_fit">Fit to screen</string>
     <string name="resize_fill">Stretch</string>
     <string name="resize_zoom">Zoom</string>
+    <string name="resize_sixteen_by_nine">SixteenByNine</string>
```

Comment:
I dont know if these are placeholders, but if they are not, then "Widescreen 16:9" and "Square 1:1" might be more readable and consistent

Reply:
Updated them as needed